package.json

```diff
@@ -14,8 +14,12 @@
   "dependencies": {
     "nan": "^2.3.5"
   },
+  "author": "Richard Chamberlain",
```

Comment:
Since this is now a foundation project, maybe this should be `contributors` like `nan` does.